src/components/sykmeldingsgrad/Sykmeldingsgrad.tsx

```diff
@@ -89,12 +90,12 @@ export const Sykmeldingsgrad = ({
     <Panel className="blokk">
       <Systemtittel>{texts.title}</Systemtittel>
       <Normaltekst>{texts.subtitle}</Normaltekst>
-      {perioderListSortert.length > 0 && (
+      {sisteOppfolgingstilfelle && perioderListSortert.length > 0 && (
         <Normaltekst>
           {texts.tilfelleVarighet}
           {tilLesbarPeriodeMedArstall(
-            tidligsteFom(perioderListSortert),
-            senesteTom(perioderListSortert)
+            sisteOppfolgingstilfelle.start,
```

Comment:
Blir det riktig når vi brukte tidligsteFom / senesteTom her tidligere?

Reply:
I teksten skriver vi i hvert fall: `Siste tilfellet sin varighet: `

Reply:
Ok, for det siste tilfelle var tidligere `tilfelleperioderReducer.data[0]` ? Da høres det riktig ut ja :)